DSA/4.2 Queues/5. Types of Queues (Priority, Circular, Deque)/4.2.5.3_Circular_Queue_Video.md

```diff
@@ -0,0 +1,11 @@
+# Circular Queue Explanation Video
+[Youtube Link](https://youtu.be/fGJXVFIpAMs)
```

Comment:
@harpreet1602 your youtube thumbnail is nice so it'd be better if you can add it here as a link. So instead of "Youtube Link", copy-paste the picture here.

Reply:
@nimisha-yadav Is it okay now?